### DISCUSSION TITLE
I need you guys

### DISCUSSION THREAD

**Discussion by j0k3r** [serverless-heaven/serverless-webpack#1555] [Team Posts]
Hi,

I would like to release 5.4.0 since November but the lack of approval on PRs really slow be down.
There is one missing PR to approve so I can finally release that version.

Can someone have a look at https://github.com/serverless-heaven/serverless-webpack/pull/713?

Thanks 🙏

**Comment by miguel-a-calles-mba** [comment]
Sorry, @j0k3r. Thanks for keeping the project moving!

My work projects no longer use GitHub and serverless-webpack. As a result, my response times have decreased. I'll work on being more responsive.
